### PR TITLE
add jsnetworkx@0.3.4

### LINKED ISSUE #6354: [Request] Add jsnetworkx
**Library name:** jsnetworkx
**Git repository url:** https://github.com/fkling/JSNetworkX.git
**npm package url(optional):** https://www.npmjs.com/package/jsnetworkx
**License(s):** BSD
**Official homepage:** jsnetworkx.org
**Wanna say something? Leave message here:**

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity (Star : 284, Fork : 65)
- [x] Project has public repository on famous online hosting platform (hosted on npm)
- [x] Has valid tags for each versions (for git auto-update)

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by Piicksarn**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):  `https://github.com/fkling/JSNetworkX`
 * Official website (optional, not the repository):  `http://jsnetworkx.org`
 * NPM package url (optional): `https://www.npmjs.com/package/jsnetworkx`
 * GitHub / Bitbucket popularity (required): `https://github.com/fkling/JSNetworkX`
   - Count of stars: `316`
   - Count of watchers: `36`
   - Count of forks:`72`
 * NPM download stats (optional):
   - Downloads in the last day: `7`
   - Downloads in the last week: `50`
   - Downloads in the last month: `102`

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by Piicksarn** [comment]
@maruilian11, would you please help me for checking the pr? thank you!

File: ajax/libs/jsnetworkx/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "name": "Felix Kling",
+    "email": "felix.kling@gmx.net",
+    "url": "http://jsnetworkx.org/"
+  },
+  "npmName": "jsnetworkx",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "jsnetworkx*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/jsnetworkx/package.json]:
```sh
$ ls jsnetworkx/
CHANGELOG.md  LICENSE.md  node/  node_modules/  package.json  README.md  transforms/
```

**Comment by PeterDaveHello** [comment]
You forgot to mention the related issue in PR comment.

File: ajax/libs/jsnetworkx/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "version": "0.3.4",
+  "description": "A graph processing and visualization library for JavaScript (port of NetworkX for Python).",
+  "license": "BSD-3-Clause",
+  "keywords": [
+    "jsnetworkx",
+    "graph",
+    "visualization",
+    "science"
+  ],
+  "homepage": "http://felix-kling.de/JSNetworkX/",
```
Comment (maruilian11) [inline ajax/libs/jsnetworkx/package.json]: http://jsnetworkx.org/ seems better

File: ajax/libs/jsnetworkx/package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    "name": "Felix Kling",
+    "email": "felix.kling@gmx.net",
+    "url": "http://jsnetworkx.org/"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fkling/JSNetworkX.git",
+    "basePath": "",
+    "files": [
+      "jsnetworkx.js"
```
Comment (maruilian11) [inline ajax/libs/jsnetworkx/package.json]: could be .+(js|map)

**Comment by Piicksarn** [comment]
@maruilian11, i have updated the pr!
please help me for checkout the pr, thank you!

**Pull request merged**